Inconsistent Windows CI error

### Summary

Rerunning seems to fix this eventually, e.g. two failures in this action: https://github.com/stfc/janus-core/actions/runs/18105502851:
```
FAILED tests/test_phonons_cli.py::test_plot - assert 1 == 0
 +  where 1 = <Result TclError('Can\'t find a usable init.tcl in the following directories: \n    {C:\\hostedtoolcache\\windows\\Pyt...ked from within\n"uplevel #0 [list source $tclfile]"\n\n\nThis probably means that Tcl wasn\'t installed properly.\n')>.exit_code
```

### Platform

Windows

### Version

janus-core version 0.8.4

### Python version

Python 3.11/3.12

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -11,6 +11,10 @@ jobs:
       matrix:
         python-version: ["3.10","3.11","3.12"]
 
+    # Use non-interactive backend for plotting
+    env:
+      MPLBACKEND: Agg
+
     steps:
       - uses: actions/checkout@v6
 

